modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -996,13 +997,20 @@ public static String getParentName(ComposedSchema composedSchema, Map<String, Sc
                     } else {
                         LOGGER.debug("Not a parent since discriminator.propertyName is not set {}", s.get$ref());
                         // not a parent since discriminator.propertyName is not set
+                        refedParentNames.add(parentName);
```

Comment:
is the comment above still valid?

Reply:
Yes it is. Also note the same existing comment in line 961 above.

For some reason, which is not clear to me, openapi generator treats this scenario (oneOf with a single $ref, where the referenced schema has no discriminator) as inheritance, despite the lack of discriminator (which would normally mean composition and not inheritance).

My fix doesn't change this behavior. It just makes it (i believe) more correct

Reply:
actually, no. you're right, I need to clean this up.
first off, the comment duplicates the log message - obvious redundancy.
second, I think it should be exactly same message as in line 961 (currently it's kind of similar, but with diffirent flavors)

Reply:
@macjohnny sorry, I'm obviously not thinking clear tonight. Should just stop answering till tomorrow.
Generally, it looks like the same comments and messages are used in both places (here and inside `getParentName()` above). Which kind of is the intention. I'll double check tomorrow to make sure it still makes sense

Reply:
I believe the comment here is still very precise.
However, I removed the LOGGER.debug above that, which just repeated that comment.
Also renamed the variable to better reflect the intention. Hope it makes more sense now